wram.asm

```diff
@@ -32,9 +32,17 @@ wMusicID:: ; c19c
 
 wMusicBank:: ; c19e
     db
-
-    ds 14 ; TODO
-
+	
+	ds 6 ; TODO
+	
+wSoundFade:: ; c1a5
+	db
+	ds 1
+
+wNextBGM:: ; c1a7
+	db
+	ds 5
```

Comment:
Unused/unknown space should generally have an empty line before and after the `ds`.
In this case, it would be:
```
wSoundFade:: ; c1a5
	db

	ds 1

wNextBGM:: ; c1a7
	db

	ds 5

```
Or:
```
wSoundFade:: db ; c1a5

	ds 1

wNextBGM:: db ; c1a7

	ds 5

```

Reply:
Done.